_includes/master/manifests/_functions.tpl

```diff
@@ -35,12 +35,8 @@ return: |
 {{- $component := index . 0 -}}
 {{- $ctx := index . 1 -}}
 
-{{- /* The remaining lines parse versions.yml to get the tag for the "component" for the given "page.version". */ -}}
-{{- /* First, grab the latest revision release from versions.yml for the given "page.version"  */ -}}
-{{- $release := index $ctx.Values $ctx.Values.page.version | first -}}
-
 {{- /* get the specified component  */ -}}
-{{- $component := index $release.components $component -}}
+{{- $component := index $ctx.Values $component -}}
```

Comment:
I considered removing the dependency on the helm function but accessing values with special characters (in this case `/` in image names, e.g. `calico/node`) raises errors in helm. Those could be fixed by refactoring the `versions.yml` file to not use the raw image name and instead use a helm friendly name (e.g. `calico/node` -> `calicoNode`) but that would ruin backwards compatibility with versions of the docs that do not use helm templating.